daal4py/sklearn/ensemble/_forest.py

```diff
@@ -481,7 +499,9 @@ def __init__(self,
                  warm_start=False,
                  class_weight=None,
                  ccp_alpha=0.0,
-                 max_samples=None):
+                 max_samples=None,
+                 maxBins=256,
+                 minBinSize=5):

```

Comment:
probably, better to set minBinSize to 1 by default to work good on small data sets

Reply:
it was fixed